test-data/run.test

```diff
@@ -902,6 +902,31 @@ assert sum == 50.0
 assert str(from_int(10)) == '10.0'
 assert str(to_int(3.14)) == '3'
 
+[case testBytes]
+def f(x: bytes) -> bytes:
+    return x
+
+def concat(a: bytes, b: bytes) -> bytes:
+    return a + b
+
+def eq(a: bytes, b: bytes) -> bool:
+    return a == b
+
+def neq(a: bytes, b: bytes) -> bool:
+    return a != b
+
+def join() -> bytes:
+    seq = (b'1', b'2', b'3')
+    return b'.'.join(seq)
```

Comment:
Could you add a bytes literal with escapes that fall outside the ASCII range?

Reply:
Sure!